buildSrc/src/main/groovy/license-report-project.gradle

```diff
@@ -80,8 +81,10 @@ class MarkdownReportRenderer implements ReportRenderer {
         project = data.project
         config = project.licenseReport
         output = new File(config.outputDir, fileName)
+        final publishExtension = project.rootProject.extensions.findByType(PublishExtension.class)
+        final prefix = (publishExtension?.spinePrefix?.getOrElse(false) ?: false) ? "spine-" : ""
```

Comment:
I'm sorry, but this is way too much...

Reply:
Here's a PR to address this: https://github.com/SpineEventEngine/config/pull/242.

Reply:
This is being addressed in https://github.com/SpineEventEngine/config/pull/243.